pyproject.toml

```diff
@@ -1,10 +1,9 @@
 [project]
```

Comment:
where is your `ruff` config? it is usually in this toml file...

Reply:
ruff didn't require any config so far, will add when it does

Reply:
there's no strict mode in ruff. i can manually enable *all* checks, but that seems excessive?

Reply:
you have to manually enable all classes of checks.
it *is* excessive -- so you will have to disable some specifically that don't make sense.
i think you need to do a similar thing to mypy... otherwise you get some middle of the ground default